modules/tinymce/src/core/test/ts/browser/SelectionOverridesTest.ts

```diff
@@ -15,9 +17,32 @@ describe('browser.tinymce.core.SelectionOverridesTest', () => {
     disable_nodechange: true,
     entities: 'raw',
     indent: false,
+    content_style: 'body { margin: 16px; }',
     base_url: '/project/tinymce/js/tinymce'
   }, [ Theme ]);
 
+  const getScrollTop = (editor: Editor) => Scroll.get(TinyDom.document(editor)).top;
+
+  const assertSelectionIsCaretBlock = (editor: Editor, caretValue: 'before' | 'after') => {
+    const selectedNode = editor.selection.getNode();
+    assert.isTrue(isCaretContainerBlock(selectedNode));
+    assert.equal(selectedNode.getAttribute('data-mce-caret'), caretValue);
+  };
+
+  const selectBesideContentEditable = (editor: Editor, contentEditableElm: HTMLElement, clickBefore: boolean) => {
+    editor.selection.scrollIntoView(contentEditableElm);
+
+    const scrollTop = getScrollTop(editor);
+    const bodyPaddingOffset = 8; // Default is 16 so divide by 2
```

Comment:
Should we call this `bodyMarginOffset` just so that it makes a bit more sense with the CSS?

Reply:
Yep, forgot to change the variable name. Thanks.